KubeArmor/core/unorchestratedUpdates.go

```diff
@@ -758,18 +759,19 @@ func (dm *KubeArmorDaemon) ParseAndUpdateContainerSecurityPolicy(event tp.K8sKub
 // backupKubeArmorHostPolicy Function
 func (dm *KubeArmorDaemon) backupKubeArmorHostPolicy(policy tp.HostSecurityPolicy) {
 	// Check for "/opt/kubearmor/policies" path. If dir not found, create the same
-	if _, err := os.Stat(cfg.PolicyDir); err != nil {
-		if err = os.MkdirAll(cfg.PolicyDir, 0700); err != nil {
-			kg.Warnf("Dir creation failed for [%v]", cfg.PolicyDir)
+	if _, err := os.Stat(cfg.GlobalCfg.RestorePath); err != nil {
+		if err = os.MkdirAll(cfg.GlobalCfg.RestorePath, 0700); err != nil {
+			kg.Warnf("Dir creation failed for [%v], setting restore path to %s", cfg.GlobalCfg.RestorePath, cfg.PolicyDir)
+			cfg.GlobalCfg.RestorePath = cfg.PolicyDir
```

Comment:
If the default value for Restorepath is policydir only, can we deprecate one of them?